<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlcoCreateCoctailForm</class>
 <widget class="QDialog" name="AlcoCreateCoctailForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Создать коктейль</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="Btn_addItem">
       <property name="text">
        <string>Добавить ингридиент</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="table"/>
     </item>
     <item>
      <widget class="QPushButton" name="Btn_Save">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="Lay_name">
       <item>
        <widget class="QLabel" name="D_name">
         <property name="text">
          <string>Название: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="W_name"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="Lay_typeCoctail">
       <item>
        <widget class="QLabel" name="D_typeCoctail">
         <property name="text">
          <string>Тип коктейля</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="W_typeCoctail"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="D_about">
       <property name="text">
        <string>Описание</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
